src/main/java/com/booster/server/NoteService.java

```diff
@@ -0,0 +1,24 @@
+package com.booster.server;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
```

Comment:
Currently, `@Slf4j` is not used, let's get rid of it.

Reply:
Done